### PR TITLE
Redirect the external storage configuration to the relevant occ commands

### LINKED ISSUE #2301: Combine External Storage Configuration and External Storage Configuration GUI documentation
Looking at the documentation in "_External Storage Configuration_", I don't see any sense it remaining a standalone file. It could be merged into "_External Storage Configuration GUI_" documentation as a NOTE admonition.

## What Type Of Content Change Is This? (Optional)
- [ ] New Content Addition
- [X] Old Content Deprecation
- [X] Existing Content Simplification
- [ ] Bug Fix to Existing Content

## Which Manual Does This Relate To? (Optional)
- [X] Admin Manual
- [ ] Developer Manual
- [ ] User Manual
- [ ] Android
- [ ] iOS
- [ ] Branded Clients
- [ ] Desktop Client
- [ ] Other

### PR DESCRIPTION
**Pull request by settermjd**
This redirect is being added as the external configuration based to a text file was deprecated in 9.0. secondly, the text said to use the relevant occ commands. So a redirect is now in place to take the users directly there. This fixes #2301.

### CODE REVIEW

**Comment by mmattel** [comment]
Just help me to understand. You redirect any request from this page to the occ commands page.
This opens the top of occ. Could a page alias be extended to redirect to a bookmark like `#files-external` to be more specific for this redirect?

**Comment by mmattel** [comment]
What is causing this PR to fail in drone?

**Comment by settermjd** [comment]
Sorry for the lack of clarity in the PR's description. Nothing is failing in Drone. The intent of this PR is to deprecate the external storage configuration file documentation. The documentation starts off with "Starting with ownCloud 9.0,...". To me, that makes it an excellent candidate for deprecation, as the current release is 10.3.2, soon to be 10.4.0. As the OCC commands are available, it makes sense, then, to redirect the user to them. Unfortunately, at least currently, page redirects do not support page anchors. Otherwise, I'd have created a redirect with an anchor in this PR.

**Comment by settermjd** [comment]
It's great to see that [the page-aliases documentation](https://docs.antora.org/antora/2.2/page/page-aliases/) is now available in the 2.2 version of the Antora docs - even if a subsequent MR changed a lot of the original content that I submitted.

**Comment by mmattel** [comment]
@settermjd ping ?

**Comment by settermjd** [comment]
> @settermjd ping ?

Thanks for the approval.

**Comment by mmattel** [comment]
What is necessary to make the checks green or how to merge ?

**Comment by settermjd** [comment]
> What is necessary to make the checks green or how to merge ?

I'm currently debugging why the build fails. At first, I thought I'd resolved the situation locally, after removing the cache directory and running the antora validation process again. Unfortunately, the validation process completes with no error output, but still returns an exit code of 1. Trying to get some help in the Antora forums to resolve the issue. After it's resolved, then the PR can be merged and backported.

**Comment by settermjd** [comment]
Actually, on restarting the Drone build, it seems like the validation step will complete successfully. Nope, didn't. But at least it provided more information this time; though not that much.

**Comment by settermjd** [comment]
I found the problem. The earlier commit created a circular redirect reference. Unfortunately, the validate script doesn't display the error, only a full antora generate does that. I've now corrected it and deprecated the old documentation

File: modules/admin_manual/pages/configuration/server/occ_command.adoc [hunk 1]
```diff
@@ -1,7 +1,7 @@
 = Using the occ Command
 :toc: macro
 :toclevels: 2
-:page-aliases: configuration/server/occ_app_command.adoc,go/admin-cli-upgrade.adoc
+:page-aliases: configuration/server/occ_app_command.adoc,go/admin-cli-upgrade.adoc,configuration/files/external_storage_configuration.adoc
```
Comment (mmattel) [inline modules/admin_manual/pages/configuration/server/occ_command.adoc]: Just for my knowledge, what is the `go` in `configuration/server/occ_app_command.adoc,go` good for?

Comment (settermjd) [reply modules/admin_manual/pages/configuration/server/occ_command.adoc]: That is for redirects from ownCloud core to the documentation. I don't have an example to hand, unfortunately.

Comment (mmattel) [reply modules/admin_manual/pages/configuration/server/occ_command.adoc]: Maybe it reads `go/admin...` but it really looks very strange

Comment (settermjd) [reply modules/admin_manual/pages/configuration/server/occ_command.adoc]: Just confirming that this redirect is correct, @mmattel

**Pull request merged**